queries.sql

```diff
@@ -1,3 +1,21 @@
-/*Queries that provide answers to the questions from all projects.*/
+/* Queries that provide answers to the questions from all projects. */
 
-SELECT * from animals WHERE name = 'Luna';
+/*
+  * Find all animals whose name ends in "mon".
+  * List the name of all animals born between 2016 and 2019.
+  * List the name of all animals that are neutered and have less than 3 escape attempts.
+  * List date of birth of all animals named either "Agumon" or "Pikachu".
+  * List name and escape attempts of animals that weigh more than 10.5kg.
+  * Find all animals that are neutered.
+  * Find all animals not named Gabumon.
+  * Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
+*/
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused code in order to have a clean code-base.